platform.txt

```diff
@@ -112,8 +112,8 @@ recipe.output.save_file_hexu={build.project_name}.save.hex
 
 ## Compute size
 recipe.size.pattern="{compiler.path}{compiler.size.cmd}" -A "{build.path}/{build.project_name}.elf"
-recipe.size.regex=\.text\s+([0-9]+).*
-recipe.size.regex.data=(\.data|\.bss|\.stack)\s+([0-9]+).*
+recipe.size.regex=(?:\.text|.\data)\s+([0-9]+).*
```

Comment:
thanks for the PR, It is probably even better if you could add the `^` (size and datasize) so that we are exactly the same as Arduino samd core upstream here 
https://github.com/arduino/ArduinoCore-samd/blob/master/platform.txt#L120